cmd/traffic/cmd/manager/internal/watchable/generic_test.tmpl.go

```diff
@@ -4,392 +4,391 @@
 package watchable_test
 
 import (
-	"context"
-	"encoding/json"
-	"testing"
-	"time"
+    "context"
+    "encoding/json"
+    "testing"
+    "time"
 
-	"github.com/stretchr/testify/assert"
+    "github.com/stretchr/testify/assert"
 
-	"VALPKG"
```

Comment:
Also incorrect. Please remove this change.

Reply:
On it.  I am watching the tests too.